api/appendix_h.asciidoc

```diff
@@ -164,6 +164,22 @@ When Device-Side Enqueue is not supported:
 
 |====
 
+When Device-Side Enqueue is supported but a replaceable On-Device Queue is not supported:
+
+[cols="2,3",options="header",]
+|====
+|*API*
+|*Behavior*
+
+| {clGetDeviceInfo}, passing +
+{CL_DEVICE_DEVICE_ENQUEUE_CAPABILITIES}
+| May not set {CL_DEVICE_QUEUE_REPLACEABLE_DEFAULT} , indicating that _device_ does not support a replaceable default On-Device Queue.
+
+| {clSetDefaultDeviceCommandQueue}
+| Returns {CL_INVALID_OPERATION} if _device_ does not support a replaceable default On-Device Queue.
```

Comment:
Consider adding this case to the list of error conditions for clSetDefaultDeviceCommandQueue.  The current error is:

> CL_​INVALID_​OPERATION if device does not support On-Device Queues.

Reply:
Thanks for catching this, added to error conditions.